internal/errs2/sanatizer.go

```diff
@@ -0,0 +1,49 @@
+package errs2
+
+import (
+	"github.com/zeebo/errs"
+	"go.uber.org/zap"
+
+	"storj.io/storj/pkg/rpc/rpcstatus"
+)
+
+type CodeMap map[*errs.Class]rpcstatus.StatusCode
+
+// LoggingSanitizer consolidates logging of original errors with sanitization of internal errors.
+type LoggingSanitizer struct {
+	wrapper *errs.Class
+	log     *zap.Logger
+	codeMap CodeMap
+}
+
+// NewLoggingSanitizer creates a new LoggingSanitizer.
+func NewLoggingSanitizer(wrapper *errs.Class, log *zap.Logger, codeMap CodeMap) *LoggingSanitizer {
+	return &LoggingSanitizer{
+		wrapper: wrapper,
+		log:     log,
+		codeMap: codeMap,
+	}
+}
+
+// Error logs the message and error to the logger and returns the sanitized error.
```

Comment:
I don't understand this sentence, is there a typo?

Reply:
I don't think so. Rephrased: `This method logs the message and the error to the logger. It returns the sanitized error.`

Reply:
That's clear to me.
After reading it, I see that the other was good, but before I didn't clearly understand it.